test/basic.html

```diff
@@ -350,13 +350,20 @@
           expect(customField.inputs[1]).to.equal(timePicker);
         });
 
-        it('custom field should not have has-value by default', () => {
+        it('should not have has-value on custom field by default', () => {
           expect(customField.hasAttribute('has-value')).to.be.false;
         });
 
-        it('custom field should have has-value when value is provided', () => {
+        it('should not have has-value on custom field if only date or time is selected', () => {
           timePicker.value = '15:00';
           dateTimePicker.__triggerCustomFieldValueUpdate();
+          expect(customField.hasAttribute('has-value')).to.be.false;
+        });
```

Comment:
Test says "if only date or time is selected" but doesn't test date (only time). I'd probably change the text here to "if only time is selected" and then copy the test and test with only date in the other test so both cases are tested.

Reply:
New test added, wording updated.